completion_aggregator/api/v1/views.py

```diff
@@ -384,22 +385,23 @@ def get(self, request, course_key):
         root_block = request.query_params.get('root_block')
         if root_block:
             root_block = UsageKey.from_string(root_block).map_into_course(course_key)
-        if is_stale:
+
+        if is_stale and waffle.flag_is_active(request, 'completion_aggregator.aggregate_stale_from_scratch'):
```

Comment:
I think it might improve performance to get the real `aggregator_queryset` here, even when `is_stale` is `True`, but testing it locally gives wildly different results, so I want to be able to try it with both configurations in the staging environment (or even production) before getting rid of this conditional.   Plus we've been talking with McKinsey about using feature flags more on big projects, so it will be nice to start building up their confidence in them with something small and simple.

Reply:
A switch would probably be sufficient, but my thought is to use a flag for flexibility unless something makes flags untenable.  (For instance, we couldn't easily use a flag for `completion.enable_completion_tracking` because a lot of the places we wanted to use it did not have access to a django request object.)

A constant is a good idea.